host/sgx/report.c

```diff
@@ -249,13 +262,13 @@ oe_result_t oe_verify_report_internal(
     }
     else if (header->report_type == OE_REPORT_TYPE_SGX_LOCAL)
     {
-        oe_result_t retval;
-
         if (enclave == NULL)
             OE_RAISE(OE_INVALID_PARAMETER);
-
+#ifndef OE_BUILD_HOST_VERIFY
```

Comment:
Passing report_type OE_REPORT_TYPE_SGX_LOCAL to host_verify package should be treated as a error. We can exclude this `else if` block and let `else` handle it.

Reply:
Done.